backend/config.js

```diff
@@ -84,12 +84,18 @@ module.exports = {
       ui_url: process.env.PATHFINDER_UI_URL,
       timeoutSeconds: process.env.API_ENDPOINT_TIMEOUT_SECONDS || 30,
     },
+    soc: {
+      url: process.env.SOC_URL || '',
+      timeoutSeconds: process.env.API_ENDPOINT_TIMEOUT_SECONDS || 30,
```

Comment:
would be better to have specific timeout overrides per endpoint, original `API_ENDPOINT` is prison api which is slow

Reply:
for example we default the offender search api below to 10 seconds instead since it is calling elastic search instead